dogs.html

```diff
@@ -0,0 +1,274 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Our Dogs</title>
+    <script src="js/dogs-javascript.js" charset="utf-8"></script>
+    <link href="styles/stylesheet.css" type="text/css" rel="stylesheet">
+</head>
+
+<body>
+
+    <header>
+        <section class="hdr-left">
+            <img class="logo" src="images/adoptadog-logo.svg" alt="Adopt a dog logo">
+            <h1>Seattle's premiere dog adoption agency</h1>
+        </section> <!-- end of hdr-left -->
+
+        <section class="hdr-right">
+            <div class="cart">
+                <img class="cart" src="images/cart-icon.svg" alt="image of a cart">
+                <p>$1234.56</p>
+            </div>
+            <!--end of cart  -->
+            <button type="submit" value="Submit" name="submit">Checkout</button>
+        </section> <!-- end of hdr-right -->
+
+
+    </header>
+
+    <nav>
+        <ul>
+            <li> <a href="./index.html">Home</a></li>
+            <li>/</li>
+            <li> <a href="./dogs.html">Dogs</a></li>
+            <li>/</li>
+            <li> <a href="./blog.html">Blog</a></li>
+
+        </ul>
+    </nav>
+    <div class="wrapper">
+        <h2>Here are all of our Dogs!</h2>
+
+        <section class="all-dogs">
```

Comment:
Right now your dog tiles seem to spill off of the page. You're definitely on the right track here with CSS grid though. Check out this article and see if this helps. Using some shortcuts, you shouldn't need to actually create col-1 row-1 CSS classes. I think this might also solve the grid overflowing issue, and make your code more responsive: https://css-tricks.com/look-ma-no-media-queries-responsive-layouts-using-css-grid/#layout-2-full-width-image-gallery